File: program/find-sum-of-n-natural-numbers-using-formula/FindSumOfNNaturalNumbersUsingFormula.rb
def sumOfNaturalNumber(num)
  num * (num+1)/2
end

if $PROGRAM_NAME == __FILE__
  print 'Please enter a Number : '
  num = gets.chomp.to_i

  puts "Sum of Natural Number : #{sumOfNaturalNumber(num)}"
end

File: program/find-sum-of-n-natural-numbers-using-formula/find_sum_of_n_natural_numbers_using_formula.pl
print "Enter a number: ";
my $num = <STDIN> ;

my $res = sumOfNaturalNumber($num);
print "Sum of natural number : $res";

sub sumOfNaturalNumber
{
    my ($x) = @_;
    my $res = $num * ($num+1)/2;
    return $res ;   
}
